interface/src/components/navbar.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react'
+import './stylesheets/navbar.css'
+
+const navbar= ()=>{
+
+    const Responsive= ()=>{
+        var x = document.getElementById("Link");
+        if (x.className === "link") {
+        x.className += " responsive";
+        x.style.display="flex !important;";
```

Comment:
Why isn't this display style part of the "Link" element CSS? The bar may change its elements spacing when the user clicks it & it may be better to start out with correct spacing.

Reply:
Thanks! that's a good point, will surely look into it.